content/testing-yaml/testing.md

```diff
@@ -21,39 +23,71 @@ npm test
 For non-UI tests or unit testing:
 
 ```bash
-./gradlew test
+- name: Test
+  script: ./gradlew test
+  test_report: app/build/test-results/**/*.xml
 ```
 
 UI tests (also known as instrumented tests):
 
 ```bash
-./gradlew connectedAndroidTest
+- name: Launch emulator
+  script: |
+    cd $ANDROID_HOME/tools
+    emulator -avd emulator &
+    adb wait-for-device
+- name: Test
+  script: |
+    set -e
+    ./gradlew connectedAndroidTest
+    adb logcat -d > emulator.log
+  test_report: app/build/outputs/androidTest-results/connected/*.xml
 ```
 
-## Native iOS
+**Tip:**: you can save the emulator log with `adb logcat -d > emulator.log` command.
+
+Instrumentation test with Firbase test lab
```

Comment:
Is Firebase test lab support only for native Android?

Reply:
In any case I would prefer formatting all the test types as headings and unifying how we call them (Unit tests, UI tests on emulator, UI tests in Firebase Test Lab). The explanation of UI and instrumentation tests could come after the heading.

Reply:
ios is possible as well, will better delete it and add with both ios and android some other time